packages/frontend-2/components/settings/workspaces/security/DefaultSeat.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <section class="flex flex-col space-y-3">
+    <div class="flex flex-col sm:flex-row gap-y-3 sm:items-center">
+      <div class="flex-1 flex-col pr-6 gap-y-1">
+        <p class="text-body-xs font-medium text-foreground">
+          Default seat for new members
+        </p>
+        <p class="text-body-2xs text-foreground-2 leading-5 max-w-[250px]">
+          Set the default seat type assigned to new workspace members.
+        </p>
+      </div>
+      <FormSelectBase
+        v-model="seatTypeModel"
```

Comment:
this should be a new vue component for sure, but i've already created one in my Invite seat type PR so we can swap this out when thats in